common/constants.go

```diff
@@ -13,16 +14,36 @@ const (
 var LoadProcess sync.Once
 var version Version
 var (
-	MAJOR = "0"
-	MINOR = "0"
-	PATCH = "0"
+	MAJOR     = "0"
+	MINOR     = "0"
+	PATCH     = "0"
+	COMMIT    = ""
+	BUILDDATE = ""
 )
 
 func GetAppVersion() Version {
 	LoadProcess.Do(parseAppVersion)
 	return version
 }
 
+func GetVersionNum() float64 {
+	version := GetAppVersion()
+	return float64(version.Major)*1000000.0 + float64(version.Minor)*1000.0 + float64(version.Patch)
+}
+
+func GetBuildTimestamp() float64 {
```

Comment:
Do these really need to be Exported from the `common` package? I could see them staying unexported in metrics.go instead...

Reply:
True, I put them there just to keep them together with `GetAppVersion` but you're right, they are only used in `metrics.go`. I'll fix it.